config/owasp/suppressions.xml

```diff
@@ -22,10 +22,12 @@
       CVE-2023-20862
       CVE-2023-20863
       CVE-2023-20873
+      CVE-2023-24998
     </notes>
     <cve>CVE-2023-20862</cve>
     <cve>CVE-2023-20863</cve>
     <cve>CVE-2023-20873</cve>
+    <cve>CVE-2023-24998</cve>
```

Comment:
Check these again after updating branch with changes from main branch.  Hopefully all of them have been resolved.